src/api/bitswap/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = (arg) => {
+  return {
+    wantlist: require('./wantlist')(arg),
+    stat: require('./stat')(arg),
+    unwant: require('./unwant')(arg)
+  }
+}
```

Comment:
This needs to be bubbled up.

`src/bitswap/index.js` and not `src/api/bitswap/index.js`